libs/traits/Cargo.toml

```
@@ -43,6 +44,7 @@ std = [
   "sp-std/std",
   "cfg-primitives/std",
   "scale-info/std",
+  "serde/std",
```

Since it's optional, shouldn't we apply `?`
```suggestion
  "serde?/std",
```

---

Wondering why adding this broke tests?

---

I think:
- `serde?/std` adds `serde/std` in case `serde` is enable
- `serde/srd` enables `serde` and adds `serde/std`